relnote(117): Measuring Tool can be controlled using the keyboard (#28764)
mdn/content: files/en-us/mozilla/firefox/releases/117/index.md

> ### Developer Tools

Reviewer: ```suggestion
### Developer tools
```

> - The area selected by the [Measuring Tool](https://firefox-source-docs.mozilla.org/devtools-user/measure_a_portion_of_the_page/index.html) can now be resized using the keyboard.
>   Pressing arrow keys moves the selected area, and <kbd>Ctrl</kbd> + arrow keys (<kbd>Cmd</kbd> + arrow keys on a Mac) resizes the selected area.
>   Holding <kbd>Shift</kbd> accelerates moving and resizing actions made with these key combinations ([Firefox bug 1262782](https://bugzil.la/1262782)).

Reviewer: minor reword suggestions

```suggestion
- The area selected by the [Measuring Tool](https://firefox-source-docs.mozilla.org/devtools-user/measure_a_portion_of_the_page/index.html) can now be resized and moved using keyboard shortcuts.
  Pressing the arrow keys moves the selected area, while pressing <kbd>Ctrl</kbd> + arrow keys (or <kbd>Cmd</kbd> + arrow keys on a Mac) resizes the selected area.
  Holding down the <kbd>Shift</kbd> key accelerates the moving and resizing actions when using these key combinations ([Firefox bug 1262782](https://bugzil.la/1262782)).
```

Author: super tnx
